frontend/js/src/user/UserFeedback.tsx

```diff
@@ -175,32 +175,30 @@ export default class UserFeedback extends React.Component<
       recordingMbidFeedbackMap,
     } = this.state;
 
-    let recording_msids = "";
-    let recording_mbids = "";
+    let recording_msids: string[] = [];
+    let recording_mbids: string[] = [];
     if (feedback?.length && currentUser?.name) {
       recording_msids = feedback
         .map((item) => item.recording_msid)
         // Only request non-undefined and non-empty string
         .filter(Boolean)
         // Only request feedback we don't already have
-        .filter((msid) => !has(recordingMsidFeedbackMap, msid))
-        .join(",");
+        .filter((msid) => !has(recordingMsidFeedbackMap, msid)) as string[];
       recording_mbids = feedback
-        .map((item) => item.recording_mbid)
+        .map((item) => item.recording_mbid!)
```

Comment:
You should be able to remove this now:
```suggestion
        .map((item) => item.recording_mbid)
```

Reply:
Actually, such a small change, I'm going to commit the suggestion straight away, hope you don't mind :)
I did some testing live on test.LB and it's working fine, so I'm keen to move it forward.